File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        String str1 = String.valueOf(a) + String.valueOf(b);
        String str2 = String.valueOf(b) + String.valueOf(a);

        int num1 = Integer.parseInt(str1);
        int num2 = Integer.parseInt(str2);

        if (num1 > num2) {
            answer = num1;
        } else {
            answer = num2;
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181939. 더 크게 합치기/더 크게 합치기.py
def solution(a, b):
    answer = 0
    num1 = str(a) + str(b)
    num1 = int(num1)
    num2 = str(b) + str(a)
    num2 = int(num2)
    
    if num1 > num2:
        answer = num1
    else:
        answer = num2
    return answer